src/js/flows/ingestFiles.ts

```diff
@@ -32,9 +32,9 @@ export default (paths: string[]): Thunk<Promise<void>> => (
     validateInput(paths, dataDir, spaceNames),
     createDir(),
     createSpace(zealot, globalDispatch, clusterId),
+    setSpace(dispatch, tabId),
```

Comment:
This is a drive by improvement. When you ingest a file, it says on the page until a space is set. We were previously waiting to set the space until after we post the file to zqd. I can't see a good reason for not setting the space right after we create it. So that's what we do now. 

It results in the app responding much faster after selecting a file.